include/oxen/quic/stream.hpp

```diff
@@ -137,6 +137,9 @@ namespace oxen::quic
         size_t unsent_impl() const override;
 
       private:
+        // Called if 0-RTT early data was rejected; marks all sent data as unacked
+        void revert_stream();
```

Comment:
I'm not sure I understand the comment: wouldn't all early data always be unacked in the case where 0rtt is rejected?  Should the comment read "unsent" rather than "unacked"?

Reply:
good catch! thank you